test/arguments.test.js

```diff
@@ -1,135 +1,120 @@
-/* global beforeEach, describe, expect, test */
-
-const createReadStream = require('fs').createReadStream
-const path = require('path')
+const { deepStrictEqual, strictEqual } = require('assert')
+const { describe, it } = require('mocha')
 const rdf = { ...require('@rdfjs/data-model'), ...require('@rdfjs/dataset') }
-const Parser = require('@rdfjs/parser-n3')
-const { fromStream } = require('rdf-dataset-ext')
-const sinon = require('sinon')
+const loadDataset = require('./support/loadDataset')
 const loader = require('../arguments')
 
-const parser = new Parser()
+const dummyLoaderRegistry = {
+  load: () => undefined
+}
 
 describe('arguments loader', () => {
   const term = rdf.namedNode('http://example.com/')
-  let loaderRegistry
-
-  beforeEach(() => {
-    loaderRegistry = {
-      load: sinon.stub()
-    }
-  })
 
   describe('loading from rdf:List', () => {
-    test('should fall back to verbatim literal value', async () => {
-      // given
+    it('should fall back to verbatim literal value', async () => {
       const dataset = await loadDataset('./arguments-list.ttl')
 
-      // when
-      const result = await loader({ term, dataset }, { loaderRegistry })
+      const result = await loader({ term, dataset }, { loaderRegistry: dummyLoaderRegistry })
 
-      // then
-      expect(result).toEqual(['a', '5'])
+      deepStrictEqual(result, ['a', '5'])
     })
 
-    test('should handle boolean false, but not undefined values from loaders', async () => {
-      // given
```

Comment:
😢

Reply:
If it's about the comments: Sorry, but for me, the tests are way more readable without them, when they are reduced to only a few lines. I'm full with you to keep the code of the test itself well structured and be picky if they are not. We can break up bigger tests with support functions or, if that's too tricky, we can add the comments only for those tests, but please let's keep only the code pattern for the average tests.